IronSoftware.Drawing/IronSoftware.Drawing.Common/Color.cs

```diff
@@ -1251,6 +1251,15 @@ public override int GetHashCode()
             return 0;
         }
 
+        /// <summary>
+        /// Translates the specified Color structure to an HTML string color representation.
+        /// </summary>
+        /// <returns>A string containing the hex representation of the color in the format #RRGGBB.</returns>
+        public string ToHtml()
```

Comment:
ToHtmlCssColorCode

Reply:
or TranslateToHtmlColor()

Reply:
Changing from ToHtml to ToHtmlCssColorCode